### ISSUE TITLE
providing prefab struct accessors creates contract error

### ISSUE THREAD

**Issue by samth** [racket/typed-racket#777]
```
#lang racket/base

(module m typed/racket/base

  (struct x ([y : Integer]) #:prefab)
  (provide x-y))

(module n racket/base
  (require (submod ".." m))
  (x-y 5))
```

produces the error:

```
; /home/samth/tmp/prefab.rkt:10:2: Type Checker: could not convert type to a contract;
;  function type has two cases of arity 1
;   identifier: x-y
;   type: (All (x) (case-> (-> (Prefab x x) x) (-> (Prefab x Any) Any)))
;   in: (x-y 5)
; Context:
;  /home/samth/sw/plt/racket/collects/syntax/wrap-modbeg.rkt:46:4 do-wrapping-module-begin
```

**Comment by pnwamk** [comment]
Quick notes from brainstorm w/ @samth : 

- goal: don't bother wrapping prefab operators in contracts; there are no invariants we wish to enforce which the functions themselves do not already enforce
- possible solution: we already store info about struct accessors/mutators/etc in a table that is tracked/serialized/etc. Perhaps we can re-use that to identify these operators and skip contract generation for them.

**Comment by pnwamk** [comment]
fixed by PR #780
